tests/test_paperqa.py

```diff
@@ -3108,3 +3108,31 @@ async def test_reader_config_propagation(stub_data_dir: Path) -> None:
     assert mock_read_doc.call_args.kwargs["chunk_chars"] == 2000
     assert mock_read_doc.call_args.kwargs["overlap"] == 50
     assert mock_read_doc.call_args.kwargs["dpi"] == 144
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("filename", ["dummy.docx", "dummy.pptx", "dummy.xlsx"])
+async def test_parse_office_doc(stub_data_dir: Path, filename: str) -> None:
+    file_path = stub_data_dir / filename
+    if not file_path.exists():
+        pytest.skip(f"{filename} not found in stub_data")
+
+    docs = Docs()
+
+    settings = Settings(
+        llm="gemini/gemini-2.5-flash",
+        embedding="gemini/text-embedding-004",
+        summary_llm="gemini/gemini-2.5-flash",
+        agent={"agent_llm": "gemini/gemini-2.5-flash"},
+        parsing=ParsingSettings(use_doc_details=False, disable_doc_valid_check=True),
+    )
+    docname = await docs.aadd(
+        file_path,
+        "dummy citation",
+        docname=filename,
+        settings=settings,
+    )
+    assert docname is not None
+    assert docs.texts
+    session = await docs.aquery("What is the RAG system?", settings=settings)
+    assert session.answer
```

Comment:
Okay I actually ran these tests just now a bit, and I noticed the question `"What is the RAG system?"` only applies to the `dummy.docx`.

Can you either:
- Adjust the question to match each document
- Change the `.pptx` and `xlsx` to also have content for `"What is the RAG system?"`

Let's make the assertions:

```python
session = await docs.aquery("What is the RAG system?", settings=settings)
assert session.used_contexts
assert len(session.answer) > 10, "Expected an answer"
assert CANNOT_ANSWER_PHRASE not in session.answer, (
    "Expected the system to be sure"
)
```